src/snowcli/cli/common/flags.py

```diff
@@ -1 +1,48 @@
+from __future__ import annotations
+
+import typer
+from snowcli.utils import check_for_connection
+
 DEFAULT_CONTEXT_SETTINGS = {"help_option_names": ["-h", "--help"]}
+
+
+ConnectionOption = typer.Option(
+    "dev",
```

Comment:
can't we use `default="dev"`? the same question for other options with `None` default value?

Reply:
> Is it only temporary as long as we use snowsql config?

Yes. We will have to polish this once configuration is managed centrally not per command basis.